supporting_content/dimensions/skupriceid.md

```diff
@@ -0,0 +1,39 @@
+## Example provider mappings 
+
+Current column mappings found in available data sets:
+
+| Provider | Data set                | Column                             |
+| ------------ | --------------------------- | -------------------------------------- |
+| AWS          | CUR                         | pricing/rate\_code \| pricing/rate\_id |
+| Azure        | Cost details export or API  | ProductID                              |
+| GCP          | BigQuery Billing Export            | sku.id                                 |
```

Comment:
GCP has no SkuPriceId equivalent.  In this example of `BigQuery Storage - Active Local Storage`, the sku.id is `947D-3B46-7781` for 2 Sku prices but there is no equivalent/exposed SkuPriceId (until we force them 😄) in GCP now

```suggestion
| GCP          | BigQuery Billing Export            | Not Available                                 |
```